metabase/metabase
# Move model cache section back to OSS

In #22892 we moved the model cache section to the enterprise folder making it go away completely in free plans. We actually want this in OSS too, so this PR moves it back to OSS closer to the question info sidebar when it's going to be used.

### To Verify

1. Run Metabase OSS
2. Go to `/admin/settings/caching` and enable model persistence
3. Spin up a sample PostgreSQL DB from [metabase/metabase-qa](https://github.com/metabase/metabase-qa) in Docker
4. Go to `/admin/databases`, connect the new PSQL DB and enable model persistence for the DB (click the "Enable model persistence" button on the DB page's sidebar)
5. Create a question using PSQL data, turn it into a model
6. Open the info sidebar (click the "info" icon at the top right)
7. You should see the caching info in the sidebar

### Demo

![CleanShot 2022-06-22 at 16 16 54@2x](https://user-images.githubusercontent.com/17258145/175067688-5e63ca28-6c12-44db-9a0b-77cb8bc1bf4c.png)
